releasenotes/notes/snmp-traps-collect-telemetry-a8dbf3ec35f2e679.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Collect telemetry metrics for SNMP Traps.
```

```suggestion
    [pkg/snmp/traps] Collect telemetry metrics for SNMP Traps.
```

To make it easier to know which part of the code this PR belongs to.